src/Bicep.Cli/Commands/BuildParamsCommand.cs

```diff
@@ -54,10 +67,49 @@ public async Task<int> RunAsync(BuildParamsArguments args)
                 return 1;
             }
 
+            var paramsOverrides = Environment.GetEnvironmentVariable("BICEP_PARAMETERS_OVERRIDES");
+
+            if (paramsOverrides is { })
+            {
+                var paramsOverridesJson = JObject.Parse(paramsOverrides);
```

Comment:
What about representing this as `Dictionary<string, JToken>` rather than `JObject`? Might make the null differentiation mentioned below more straightforward.

Reply:
Yes, that would make null differentiation straightforward. But how are we going to get the individual value `JToken` without using `JObject.parse`  to parse the entire thing first?